data/src/common_characters.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+"""Creates json file containing list of "Common" for each language
+
+This module takes TSV files from data and return a json file for 
```

Comment:
"JSON" again. 

"return" is a weird word here, I'd say it "writes" it or something like that.